querier/src/chunk/mod.rs

```diff
@@ -276,6 +369,69 @@ impl ParquetChunkAdapter {
             partition_sort_key,
         ))
     }
+
+    /// Create new cached chunk.
+    ///
+    /// Returns `None` if some data required to create this chunk is already gone from the catalog.
+    pub async fn new_chunk(
```

Comment:
```suggestion
    pub async fn new_rb_chunk(
```

Or maybe 

```suggestion
    pub async fn new_cached_chunk(
```

There are already so many "chunk"s in the system, making it a little more specific I think will help